index.js

```diff
@@ -50,8 +50,9 @@ function Nemo(_basedir, _configOverride, _cb) {
 
   //settle arguments
   if (arguments.length === 0) {
-    error('Nemo constructor needs at least a callback');
-    var noCallbackError = new Error('Nemo constructor needs at least a callback');
+    var ERROR_MESSAGE = 'Nemo constructor needs at least a callback';
```

Comment:
would prefer camelCasing here. uppercase/titlecase should be reserved for named constants.